package.json

```diff
@@ -29,17 +29,17 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.7",
-    "appium-adb": "2.0.0-beta14",
+    "appium-adb": "2.0.0-beta15",
```

Comment:
We ought to be able to use `beta` and `latest` for our packages, while they are up in the air. The packages need to be published with the tags, though (at least, the beta ones do).